assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_metrics/run.py

```diff
@@ -183,7 +183,10 @@ def compute_metrics(histogram_df, threshold_args, metric_names):
                     metadata_schema,
                 )
             aggregated_metrics_df = aggregated_metrics_df.union(threshold_row)
-        if average_score_metric_name in input_metric_names:
+        # for now, compute average score if either average score or pass rate is requested
+        compute_average = average_score_metric_name in input_metric_names
```

Comment:
how about threshold? How to calculate the threshold for the average score?

Reply:
good point, let me look into this more next